notebooks/community/sdk/sdk_pytorch_torchrun_custom_container_training_imagenet.ipynb

```diff
@@ -0,0 +1,2732 @@
+{
```

Comment:
<p>Split cell into 4: Overview, Objective, Dataset, Costs. Note that Dataset is after Objective.</p><p> </p><p>Google Cloud Storagte -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Moved section and updated Google Cloud Storage to Cloud Storage</p>